packages/envd/internal/host/mmds.go

```diff
@@ -22,17 +22,19 @@ const (
 )
 
 type MMDSOpts struct {
-	TraceID    string `json:"traceID"`
-	InstanceID string `json:"instanceID"`
-	EnvID      string `json:"envID"`
-	Address    string `json:"address"`
+	TraceID      string `json:"traceID"`
+	InstanceID   string `json:"instanceID"`
+	EnvID        string `json:"envID"`
+	Address      string `json:"address"`
+	EventProxyIP string `json:"eventProxyIP"`
```

Comment:
```suggestion
	HyperloopIP string `json:"hyperloopIP"`
```

Ignore if you have a different approach, but it's just an IP address used for pointing to a sandbox server. We don't necessarily need to be stuck forever with `eventProxyIP` and can name it `hyperloopProxyIP` / `hyperloopIP`, so people can reuse it for different purposes.

Reply:
fixed